maintainers/maintainer-list.nix

```diff
@@ -7523,16 +7523,6 @@
     githubId = 29855073;
     name = "Michael Colicchia";
   };
-  matdsoupe = {
```

Comment:
You also need to update https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/earthly/default.nix#L40